localstack/services/awslambda/invocation/docker_runtime_executor.py

```diff
@@ -24,6 +27,13 @@
 from localstack.utils.docker_utils import DOCKER_CLIENT as CONTAINER_CLIENT
 from localstack.utils.strings import truncate
 
+# Hook definitions
+HOOKS_LAMBDA_START_DOCKER_EXECUTOR = "localstack.hooks.lambda_start_docker_executor"
+HOOKS_LAMBDA_PREPARE_DOCKER_EXECUTOR = "localstack.hooks.lambda_prepare_docker_executors"
```

Comment:
Should we maybe move this to `localstack/services/awslambda/hooks.py` ? :thinking: 
Might make sense to have some sort of convention for this

cc @thrau

Reply:
Will do, makes sense!